public/bin/shellshare

```diff
@@ -31,7 +31,9 @@ try:
     from urlparse import urlparse
 except ImportError:
     from urllib.parse import urlparse
-
+if platform.system() == 'Windows':
+    import shutil
+    import requests
```

Comment:
`requests` is an external dependency, right? It may need people to install it before running shellshare. Maybe we can use `urllib.request` instead?

Reply:
It's been in every distribution I've tested, but if urllib.request is more commonly installed, that's no problem to change.